src/api-server-selector.js

```diff
@@ -220,6 +235,17 @@ export class ApiServerSelector extends EventsTargetMixin(AmfHelperMixin(LitEleme
     node.removeEventListener('api-navigation-selection-changed', this._handleNavigationChange);
   }
 
+  _notifyServersCount() {
+    const serversCount = this._getServersCount();
+    this.dispatchEvent(
+      new CustomEvent('api-servers-count-changed', {
+        detail: { serversCount },
+        bubbles: false,
```

Comment:
This relates to my comment in the other PR. This event does not bubble. BTW, `false` is the default value so there's no point of declaring this.
Please, add tests to test events behavior (I am thinking about the other component, there's no way it handles the event not registered directly on the element).